static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -450,7 +450,9 @@ export default class AudioPlayer extends Component<Props, State> {
 
     seek = (chosenPoint: number) => {
         const chosenTime = (this.audio.duration * chosenPoint) / 100;
-        this.updatePlayerTime(chosenTime);
+        const timeCheckedAgainstMax = Math.min(chosenTime, this.state.duration);
```

Comment:
Let's simplify to `const checkedTime = Math.max(0, Math.min(this.state.duration, chosenTime))`